### PR TITLE
Feature/seed data for taxonomy

### PR DESCRIPTION
**Pull request by smita7875**
Created rake task to import taxonomies data 
Added master data xlsx file to /docs/master_data folder

### CODE REVIEW

File: lib/tasks/taxonomies.rake [hunk 1]
```diff
@@ -0,0 +1,39 @@
+require 'rubyXL'
+
+namespace :taxonomies do
+  desc 'To seed data' 
+	task :import_data => :environment do
+		puts "This is my task 1 "
```
Comment (chetan-jadhav) [inline lib/tasks/taxonomies.rake]: We dont need this puts statement

File: lib/tasks/taxonomies.rake [hunk 2]
```diff
@@ -0,0 +1,39 @@
+require 'rubyXL'
+
+namespace :taxonomies do
+  desc 'To seed data'
```
Comment (chetan-jadhav) [inline lib/tasks/taxonomies.rake]: Description should be short text to tell what task is doing.
Example. "To import taxonomies data from /docs/master_data/file_name.xlsx folder into database"

File: lib/tasks/taxonomies.rake [hunk 3]
```diff
@@ -0,0 +1,39 @@
+require 'rubyXL'
+
+namespace :taxonomies do
+  desc 'To seed data' 
+	task :import_data => :environment do
+		puts "This is my task 1 "
+		workbook = RubyXL::Parser.parse "#{Rails.root}/docs/master_data/MasterData.xlsx"
```
Comment (chetan-jadhav) [inline lib/tasks/taxonomies.rake]: Please handle file not found exception here and return a meaningful message.

File: lib/tasks/taxonomies.rake (hunk 1 above)
Comment (smita7875) [reply lib/tasks/taxonomies.rake]: Removed debug statements

File: lib/tasks/taxonomies.rake (hunk 2 above)
Comment (smita7875) [reply lib/tasks/taxonomies.rake]: Updated description

File: lib/tasks/taxonomies.rake (hunk 3 above)
Comment (smita7875) [reply lib/tasks/taxonomies.rake]: Added exception handler

**Pull request merged**